firebase-firestore/src/androidTest/java/com/google/firebase/firestore/IndexingTest.java

```diff
@@ -108,9 +108,13 @@ public void testBadIndexDoesNotCrashClient() {
                     + "}"));
   }
 
+  /**
```

Comment:
Add a `TODO(b/296100693) Add testing hooks to verify indexes are created as expected` comment here, and in the comment of the following function. This will help future you or somebody else correlate the buganizer ticket with the code in question, and vice versa.